src/Configuration.php

```diff
@@ -69,13 +80,26 @@ public static function forSymmetricSigner(
             $key,
             $encoder,
             $decoder,
+            null,
+            null,
+            null,
         );
     }
 
     /** @param callable(ClaimsFormatter): Builder $builderFactory */
-    public function setBuilderFactory(callable $builderFactory): void
+    public function setBuilderFactory(callable $builderFactory): self
```

Comment:
Should we rename those setters to follow the whole `withWhateva()` convention?

Reply:
Done